### DISCUSSION TITLE
create-seia-app@0.1.1

### DISCUSSION THREAD

**Discussion by abiriadev** [abiriadev/seia#19] [Announcements]
:tada: We got a new contributor!!

welcome to seia, @aidnyu!

## What's Changed
* fix(create-seia-app/ts-template): correct `@types/react-dom` version to `^18.3.0` by @aidnyu in https://github.com/abiriadev/seia/pull/18

## New Contributors
* @aidnyu made their first contribution in https://github.com/abiriadev/seia/pull/18

**Full Changelog**: https://github.com/abiriadev/seia/compare/create-seia-app@0.1.0...create-seia-app@0.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/abiriadev/seia/releases/tag/create-seia-app%400.1.1'>create-seia-app@0.1.1</a>.</em>

**Comment by aidnyu** [comment]
Thank you for the warm welcome 😊
